### ISSUE TITLE
Biały kontener, który pojawił się przy scrollowaniu

### ISSUE THREAD

**Issue by kamilsabatowski** [SilverDecisions/SilverDecisions#183]
![zrzut ekranu 2019-01-24 o 21 13 58](https://user-images.githubusercontent.com/25226087/52239615-2bd17780-28cf-11e9-940b-873e5b29911f.png)

W załączeniu screenshot, który pokazuje, że nie da się przewinąć (lub powiększyć białej przestrzeni pod drzewem decyzyjnym) - przewijanie spowodowało, że pojawił się biały kontener bez jakiejkolwiek treści (screen). Biały obszar poniżej części interfejsu z drzewem był nieużywalny w jakikolwiek sposób. Nie dało się wykorzystać go, przewinąć drzewa, dodać trochę przestrzeni pod drzewem.

**Comment by mwasiluk** [comment]
Czy występuje to tylko przy otwartym menu kontekstowym?

**Comment by kamilsabatowski** [comment]
Nie, kilkukrotnie wystąpiło to w różnych przypadkach - raz z oknem kontekstowym, raz przy próbie zejścia poniżej drzewa.

**Comment by bkamins** [comment]
@michaljakubczyk - możesz proszę sprawdzić czy możesz to zreplicować na macOS?

**Comment by michaljakubczyk** [comment]
Udaje mi się jedynie przy otwartym menu kontekstowym. Po zamknięciu tego menu ta biała przestrzeń znika.
